core/market/decentralized/src/matcher/store.rs

```diff
@@ -58,6 +58,19 @@ impl SubscriptionStore {
             })
     }
 
+    pub async fn get_offers(&self) -> Result<Vec<ClientOffer>, OfferError> {
+        let model_offers = self
+            .db
+            .as_dao::<OfferDao>()
+            .get_offers()
+            .await
+            .map_err(|e| OfferError::GetMany(e))?;
+        Ok(model_offers
+            .into_iter()
+            .map(|o| o.into_client_offer().unwrap())
+            .collect())
```

Comment:
I think we need at least error msg when the conversion fails

```suggestion
        Ok(self
            .db
            .as_dao::<OfferDao>()
            .get_offers()
            .await
            .map_err(|e| OfferError::GetMany(e))?
            .into_iter()
            .filter_map(|o| match o.into_client_offer() {
                Err(e) => {
                    log::error!("Skipping Offer because of: {}", e);
                    None
                }
                Ok(o) => Some(o),
            })
            .collect())
```

Reply:
I'd also add
```
--- a/core/market/decentralized/src/db/models/offer.rs
+++ b/core/market/decentralized/src/db/models/offer.rs
@@ -83,8 +83,8 @@ impl Offer {
             constraints: self.constraints.clone(),
             properties: serde_json::from_str(&self.properties).map_err(|e| {
                 format!(
-                    "Can't serialize Offer properties from database. Error: {}",
-                    e
+                    "Can't serialize Offer [{}] properties from database. Error: {}",
+                    self.id, e,
                 )
             })?,
         })
```